releases/index.html

```diff
@@ -23,13 +24,59 @@
   | <a href="#14.2">14.2</a>
   | <a href="#14.1">14.1</a>
 </p>
-
 <div class="grid-10">
   <div class="version">
+    <a name="16.2"></a>
     <a name="latest"></a>
-
+    <h4>16.2 (latest)</h4>
+    <div class="section">
+      <h5>What's new in GoCD 16.2</h5>
+    <div class="section">
+      <h5>Features and bug fixes</h5>
+      <ul>
+        <li>
+          <a href="https://github.com/gocd/gocd/pull/1808">Select a range of agents via shift-clicking checkboxes</a>
+        </li>
+        <li>
+          <a href="https://github.com/gocd/gocd/pull/1909">Fix system call to kill processes on agents running JDK8</a>
+        </li>
+        <li>
+          <a href="https://github.com/gocd/gocd/pull/1898">[Security] When editing SVN, TFS, P4 materials, the plain password is printed in html.</a>
+        </li>
+        <li>
+          <a href="https://github.com/gocd/gocd/issues/871">Output environment variables once per job instead of per task</a>
+        </li>
+        <li>
+          <a href="https://github.com/gocd/gocd/pull/1822">[Performance] Turn off JRuby's invokedynamic on to improve performance on servers using JDK8</a>
+        </li>
+        <li>
+          <a href="https://github.com/gocd/gocd/issues/1785">Git material is always cloned if branch is set to empty string</a>
+        </li>
+        <li>
+          <a href="https://github.com/gocd/gocd/issues/1646">[Config] Do not allow blank users in "authorization"</a>
+        </li>
+      </ul>
+      <div class="section">
+        <h5>Note</h5>
+        <p>
+          The comprehensive list of changes for this release can be found <a href="https://github.com/gocd/gocd/pulls?q=is%3Apr+milestone%3A%22Release+16.2%22+is%3Aclosed">here.</a>
```

Comment:
`is:closed` filter should not apply, assuming that everything in the release must be closed.